Makefile

```diff
@@ -103,3 +103,21 @@ pypi-test:
 pypi-test-plugin:
 	pip install twine && \
 	python -m twine upload -u=__token__ -p=$(TEST_PYPI_API_TOKEN) --repository testpypi $(PLUGINFOLDER)/dist/*
+
+install-simple-example:
+	make install && \
+	make PLUGINFOLDER=plugins/streams/redis install-plugin && \
+	make PLUGINFOLDER=plugins/storage/fs install-plugin && \
+	make PLUGINFOLDER=plugins/ops/config-manager install-plugin && \
```

Comment:
missing:
	make PLUGINFOLDER=plugins/storage/redis install-plugin && \

Reply:
Is redis-storage needed in simple-example?

Reply:
no, it is not! :)